configuration/configuration.go

```diff
@@ -175,6 +176,9 @@ func (c *ConfigurationData) setConfigDefaults() {
 	c.v.SetDefault(varKeycloakTesUserName, defaultKeycloakTesUserName)
 	c.v.SetDefault(varKeycloakTesUserSecret, defaultKeycloakTesUserSecret)
 
+	// Max number of users returned when searching users
+	c.v.SetDefault(varUsersListLimit, 50)
```

Comment:
Effectively the number 50 should be fine.. since the UI ( collaborators' ) page doesn't require more than this.